uk.co.compucorp.civicrm.hrcontactactionsmenu/templates/CRM/HRContactActionsMenu/Page/Inline/Actions.tpl

```diff
@@ -1,69 +1,70 @@
 <div id="crm-contact-actions-wrapper" data-edit-params='{ldelim}"cid": "{$contactId}", "class_name": "CRM_Contact_Page_Inline_Actions"{rdelim}'>
   {crmButton id="crm-contact-actions-link" href="#" icon="bars"}
-  {ts}Actions{/ts}
+    {ts}Actions{/ts}
   {/crmButton}
-  <div id="crm-contact-actions-list">
+  <div id="crm-contact-actions-list" class="crm_contact-actions">
     <div id="bootstrap-theme">
-      <div class="row">
-        <div class="col-md-4 crm_contact_action_menu__user_info">
-            {foreach from=$menu->getHighlightedPanelItems() item='group'}
-            <div class="row">
+      <div class="crm_contact-actions__body">
+        <div class="col-md-4 crm_contact-actions__body__user-info crm_contact-actions__user-info--margin">
```

Comment:
what's this `crm_contact-actions__user-info--margin` class for? Please add screenshot

Reply:
I already removed its style. We don't need it . I will remove.